wolox-books/src/app/screens/auth/screens/book-list/book-list.component.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../../../../scss/colors';
+@import '../../../../../scss/fonts';
+
+$book-width: 220px;
+$book-height: 260px;
+
+.books {
+  margin: 100px 300px;
+  display: flex;
+  flex-wrap: wrap;
+
+  .book {
+    margin: 20px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    max-width: $book-width;
+    width: $book-width;
+    height: $book-height;
+    padding: 15px;
+    background-color: white;
+    box-shadow: 8px 8px 5px -5px $light-grey;
+
+    .book-title {
+      font-size: $text-medium;
+      font-weight: bold;
+    }
+
+    .book-author {
+      font-size: $text-very-small;
+    }
+
+    .book-img {
+      margin-bottom: 15px;
+      width: 100%;
+      height: 170px;
+    }
+  }
+}
```

Comment:
I think the nesting is not obligatory. When you need to override this classes, it will be more difficult if you nest them without the need